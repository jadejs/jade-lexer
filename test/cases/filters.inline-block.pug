p: :markdown-it **bolded**
p::markdown-it **bolded**
