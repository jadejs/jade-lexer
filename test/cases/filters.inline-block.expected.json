{"type":"tag","line":1,"col":1,"val":"p"}
{"type":":","line":1,"col":2}
{"type":"filter","line":1,"col":4,"val":"markdown-it"}
{"type":"text","line":1,"col":17,"val":"**bolded**"}
{"type":"newline","line":2,"col":1}
{"type":"tag","line":2,"col":1,"val":"p"}
{"type":":","line":2,"col":2}
{"type":"filter","line":2,"col":3,"val":"markdown-it"}
{"type":"text","line":2,"col":16,"val":"**bolded**"}
{"type":"newline","line":3,"col":1}
{"type":"eos","line":3,"col":1}